src/OpenCL.cpp

```diff
@@ -367,12 +468,21 @@ void OpenCL_Network::forward(const std::vector<net_t>& input,
             m_ceil * m_ceil *  max_channels * sizeof(net_t);
         const auto alloc_vm_size =
             WINOGRAD_TILE * m_ceil * n_ceil * sizeof(net_t);
+
+#ifndef NDEBUG
+        //Check that VBuffer can be used as a merge buffer for convolve1
```

Comment:
So uh, in which cases would this not be true (and hence we crash/give wrong output) in release?

Reply:
It's always enough. mergebuffer needs to be max_channels/4 planes, while Vbuffer is always at least max_channel planes. I'll remove this check.